src/app/login/LoginForm.tsx

```diff
@@ -0,0 +1,166 @@
+"use client";
+
+import {
+  Box,
+  Button,
+  Typography,
+  Avatar,
+  Dialog,
+  DialogContent,
+  DialogContentText,
+  DialogActions,
+  Snackbar,
+} from "@mui/material";
+import { useForm, FormProvider } from "react-hook-form";
+import { login } from "@/apis/auth";
+import SenifitTextField from "../../components/SenifitTextField";
+import { useRouter } from "next/navigation";
+import { useState } from "react";
+
+type LoginFormValues = {
+  id: string;
+  password: string;
+};
+
+export default function LoginForm() {
+  const methods = useForm<LoginFormValues>({ mode: "onBlur" });
```

Comment:
<img width="363" height="195" alt="스크린샷 2025-08-06 오후 3 31 56" src="https://github.com/user-attachments/assets/aebad249-8b2d-422b-9685-dc89e976ec14" />

디자인 상 `onBlur`가 아닌 `onSubmit`으로 처리되어야합니다. 공식문서는 [이쪽](https://www.react-hook-form.com/api/useform/#mode)을 참고하여주세요

```suggestion
  const methods = useForm<LoginFormValues>({ mode: "onSubmit" });
```